CHANGELOG.md

```diff
@@ -126,6 +126,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Remove `github-checks-reporter`, an unused dependency ([#3126](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3126))
 - Upgrade `vega-lite` dependency to ^5.6.0 ([#3076](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3076))
 - Bumps `re2` and `supertest` ([3018](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3018))
+- Allow relaxing the Node.js runtime version requirement ([3402](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3402))
```

Comment:
Does it allow relaxing, or does it actually relax it?
```suggestion
- Relax the Node.js runtime version requirement and parse via semver ([3402](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3402))
```

Reply:
It allows relaxing because if don't change anything, we will have `14.20.1` which will be translated to `=14.20.1` which is as strict as it gets.